src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1135,6 +1142,7 @@ public LocalDate fromString(String string) {
           textFieldNumberOfWinners.setDisable(false);
           checkBoxBatchElimination.setDisable(false);
           checkBoxContinueUntilTwoCandidatesRemain.setDisable(false);
+          checkBoxFirstRoundDeterminesThreshold.setDisable(false);
```

Comment:
The note in the txt file seems to imply that it only works for single-winner. Doesn't that conflict with this?

Reply:
It theoretically works with sequential winner-takes-all, but doesn't make any political sense and isn't needed by anybody.

I'll remove this to make it match the notes.